codecov.skip

```diff
@@ -0,0 +1,3 @@
+istio.io/istio/tests*
+istio.io/istio/mixer/tools/codegen*
+istio.io/istio/security/tests/integration*
```

Comment:
why not those?

Reply:
the first 2 were already skipped, I added the integration one because they need a real cluster.